pyproject.toml

```diff
@@ -0,0 +1,6 @@
+[build-system]
```

Comment:
Specifies *build tools* and *build requirements*. (<https://packaging.python.org/tutorials/packaging-projects/#creating-pyproject-toml>)